### PR TITLE
Update plugin_transmission.py

### PR DESCRIPTION
**Pull request by tvcsantos**
Added a new config option to remove torrents based on their download directory.

Useful when combined with automatic downloads when you don't know in advance the trackers from where they are downloaded. For example when downloading torrents from a showrrs.info.

### CODE REVIEW

File: flexget/plugins/plugin_transmission.py [hunk 1]
```diff
@@ -660,6 +662,7 @@ def on_task_exit(self, task, config):
         delete_files = bool(config['delete_files']) if 'delete_files' in config else False
         trans_checks = bool(config['transmission_seed_limits']) if 'transmission_seed_limits' in config else False
         tracker_re = re.compile(config['tracker'], re.IGNORECASE) if 'tracker' in config else None
+        directories_re = config['directories'] if 'directories' in config else None
```
Comment (liiight) [inline flexget/plugins/plugin_transmission.py]: Can't you just use:
```python
directories_re = config.get('directories')
```
Also, do you need to update the schema to include this option?

Comment (tvcsantos) [inline flexget/plugins/plugin_transmission.py]: Done. No need to update the schema, It's optional

**Comment by alneven** [comment]
do you have an example? as this is a very good idea!

**Comment by tvcsantos** [comment]
```
transmission_cleanup:
   priority: 5
   seen: local
   clean_transmission:
      username: YOUR_USER
      password: YOUR_PASS
      finished_for: 2 hours
      directories:
            - \A/+media/+beta/+Downloads/+Shows/+Auto(\Z|/+.*)
```

This will remove all torrents under /media/beta/Downloads/Shows/Auto and its subdirectories

**Comment by tvcsantos** [comment]
@alneven 

the list under directories is a list of regex. For more details on the regex that i've mentioned go to https://regex101.com/#python

File: flexget/plugins/plugin_transmission.py (hunk 1 above)
Comment (liiight) [inline flexget/plugins/plugin_transmission.py]: I apologize if I'm missing something but the schema has the key `'additionalProperties': False`. How can you add `directories` to it?

**Comment by alneven** [comment]
Nice, thank you!

**Comment by gazpachoking** [comment]
Thanks! Can you document this on the plugin wiki page?

**Comment by tvcsantos** [comment]
@gazpachoking Done, thanks.

**Pull request merged**